linera-views/src/scylla_db.rs

```diff
@@ -5,45 +5,57 @@
 //! The code is functional but some aspects are missing.
 //!
 //! The current connection is done via a Session and a corresponding
-//! primary key that we name `namespace`. The maximum number of
+//! primary key that we name `table_name`. The maximum number of
 //! concurrent queries is controlled by max_concurrent_queries.
 //!
+//! We thus implement the
+//! * [`KeyValueStoreClient`][trait1] for a database access.
+//! * [`Context`][trait2] for the context.
+//!
 //! Support for ScyllaDb is experimental and is still missing important features:
```

Comment:
```suggestion
//! Support for ScyllaDB is experimental and is still missing important features:
```

Also below. (In plain text; identifiers should remain `ScyllaDb`.)

Reply:
Ok, done.